doc/source/whatsnew/v0.21.1.txt

```diff
@@ -102,7 +102,7 @@ Sparse
 Reshaping
 ^^^^^^^^^
 
--
+- Error message in ``pd.merge_asof()`` for key datatype mismatch now includes datatype of left and right key (:issue:`18068`)
```

Comment:
can you use `` :func:`merge_asof` `` so we get the link

Reply:
@jreback Is there a documentation for syntax to be used for release notes? Also, how can I view rendered version of it?

Making the required change. Also, will add test for the same.

Reply:
no this is fine

Reply:
documentation is look at similar notes!

Reply:
you would have to build the docs to render it